scaper/core.py

```diff
@@ -91,6 +101,35 @@ def generate_from_jams(jams_infile, audio_outfile, fg_path=None, bg_path=None,
     isolated_events_path : str
         Path to folder for saving isolated events. If None, defaults to
         `<audio_outfile parent folder>/<audio_outfile name>_events`.
+    disable_sox_warnings : bool
+            When True (default), warnings from the pysox module are suppressed
+            unless their level is ``'CRITICAL'``. If you're experiencing issues related
+            to audio I/O setting this parameter to False may help with debugging.
+    txt_path: str or None
+            Path for saving a simplified annotation in a space separated format
+            [onset  offset  label]. Good for loading labels in e.g. Audacity. If
+            None, does not save txt annotation to disk.
+    txt_sep: str
+        The separator to use when saving a simplified annotation as a text
+        file (default is tab for compatibility with Audacity label files).
+        Only relevant if txt_path is not None.
+
+    Returns
+    -------
+    soundscape_audio : np.ndarray
+        The audio samples of the generated soundscape. Returns None if
+        no_audio=True.
+    soundscape_jam: jams.JAMS
+        The JAMS object containing the full soundscape annotation.
+    annotation_list : list
+        A simplified annotation in a space-separated format
+        [onset  offset  label].
+    event_audio_list: list
+        A list of np.ndarrays containing the audio samples of every
+        individual background and foreground sound event. Events are listed
+        in the same order in which they appear in the jams annotations data
+        list, and can be matched with:
+        `for obs, event_audio in zip(ann.data, event_audio_list): ...`.
```

Comment:
I think we should add an example from the script that uses the return API once we have it.

Reply:
I totally agree we need examples! Since we're missing them everywhere though, perhaps we try and squeeze this PR through quickly first, and then make a new one to add examples everywhere? A docs PR!

Reply:
Sounds good to me.